.github/actions/setup-edge-linux/setup-edge.ps1

```diff
@@ -0,0 +1,13 @@
+function Program {
```

Comment:
What's the reason for wrapping it into a function? We don't run anything after it.